mechanical/assembly_renderer.py

```diff
@@ -19,14 +19,17 @@
 from pathlib import Path
 import cadquery as cq
 import yaml
-
-from nimble_build_system.cad.shelf import create_shelf_for
+import logging
+from nimble_build_system.cad.shelf import create_shelf_for_x
 
 from nimble_build_system.cad.rack_assembly import RackAssembly
+# from nimble_build_system.orchestration.paths import ABS_PATH
 
 assembly_definition_file = "../build/assembly-def.yaml"
 render_destination = os.path.join(os.getcwd(), "renders")
 
+logging.info(f"RENDER: {render_destination}")
```

Comment:
Would be good to use `LOGGER = getLogger(__name__)` after import and then use this logger rather than the root logger.

This applies to a number of places through the code.